components/FeatureHighlight/tests/unit/FeatureHighlightTitleBodyAccessibilityMutatorTests.m

```diff
@@ -0,0 +1,120 @@
+    [MDCFeatureHighlightAccessibilityMutator
+        mutateTitleColorForFeatureHighlightViewController:hfvc
+                             withTextAccessibilityOptions:options];
+    [MDCFeatureHighlightAccessibilityMutator
+         mutateBodyColorForFeatureHighlightViewController:hfvc
+                             withTextAccessibilityOptions:options];
+
+    XCTAssertNotNil(hfv.titleColor);
+    XCTAssertNotNil(hfv.bodyColor);
+    XCTAssertNotEqualObjects(hfv.titleColor, color, @"");
+    XCTAssertNotEqualObjects(hfv.bodyColor, color, @"");
+  }
+}
+
+- (void)testMutatorKeepsAccessibleTextColor {
+  NSDictionary* colors = @{ [UIColor redColor]: [UIColor blackColor]};
+  for (UIColor *color in colors) {
+    MDCFeatureHighlightViewController *hfvc =
+        [[MDCFeatureHighlightViewController alloc] initWithHighlightedView:_highlightedView
+                                                               andShowView:_showView
+                                                                completion:nil];
+    MDCFeatureHighlightView *hfv = (MDCFeatureHighlightView *)hfvc.view;
+
+    // Making the background color accessible with title/body color.
+    hfv.outerHighlightColor = colors[color];
+    hfv.titleColor = color;
+    hfv.bodyColor = color;
+
+    MDFTextAccessibilityOptions options = MDFTextAccessibilityOptionsPreferLighter;
+    [MDCFeatureHighlightAccessibilityMutator
+        mutateTitleColorForFeatureHighlightViewController:hfvc
+                             withTextAccessibilityOptions:options];
+    [MDCFeatureHighlightAccessibilityMutator
+        mutateBodyColorForFeatureHighlightViewController:hfvc
+                            withTextAccessibilityOptions:options];
+
+    XCTAssertEqualObjects(hfv.titleColor, color, @"");
+    XCTAssertEqualObjects(hfv.bodyColor, color, @"");
+  }
+}
+
+- (void)testMutatorSelectsTheRightColorWhenThereIsNoColorSet {
+  MDCFeatureHighlightViewController *hfvc =
+      [[MDCFeatureHighlightViewController alloc] initWithHighlightedView:_highlightedView
+                                                             andShowView:_showView
+                                                              completion:nil];
+  MDCFeatureHighlightView *hfv = (MDCFeatureHighlightView *)hfvc.view;
+
+  // Making the background color accessible with title/body color.
+  hfv.outerHighlightColor = [UIColor blackColor];
+
+  MDFTextAccessibilityOptions options = MDFTextAccessibilityOptionsPreferLighter;
+  [MDCFeatureHighlightAccessibilityMutator
+   mutateTitleColorForFeatureHighlightViewController:hfvc
+   withTextAccessibilityOptions:options];
+  [MDCFeatureHighlightAccessibilityMutator
+   mutateBodyColorForFeatureHighlightViewController:hfvc
+   withTextAccessibilityOptions:options];
+
+  XCTAssertNotNil(hfv.titleColor);
+  XCTAssertNotNil(hfv.bodyColor);
+  XCTAssertNotEqualObjects(hfv.titleColor, [UIColor blackColor], @"");
```

Comment:
Please verify that the expected color is set. This seems to only check that it isn't `blackColor`.

Reply:
Already replied to this on your first PR.